packages/react-dev-overlay/src/internal/components/Overlay/maintain--tab-focus.ts

```diff
@@ -804,11 +804,16 @@ var focusSummary = {
 }
 
 function makeFocusableForeignObject() {
-  var fragment = document.createElement('div')
-  fragment.innerHTML =
-    '<svg><foreignObject width="30" height="30">\n      <input type="text"/>\n  </foreignObject></svg>'
+  var foreignObject = document.createElementNS(
+    'http://www.w3.org/2000/svg',
+    'foreignObject'
+  )
+  foreignObject.setAttribute('width', 30)
```

Comment:
Instead of using `setAttribute`, you can use the standard `SVGAnimatedLength` API:

```
foreignObject.width.baseVal.value = 30;
```

Reply:
Fixed